testomania/src/main/java/com/earth/testomania/presentation/home/HomeScreen.kt

```diff
@@ -53,7 +57,11 @@ fun HomeScreen(
     }
 
     BottomSheetScaffold(
-        modifier = Modifier.systemBarsPadding(),
+        modifier = Modifier
+            .systemBarsPadding()
+            .pointerInput(Unit) {
```

Comment:
how does BottomSheetScaffold know where exactly is bottomsheet? I would expect that pointerInput() would always trigger...

Reply:
hmmm yes, you are right. i will change that